DigitalLearningSolutions.Web/Views/NotificationPreferences/UpdateNotificationPreferences.cshtml

```diff
@@ -17,40 +17,41 @@
   }
 }
 
-<form method="post" asp-controller="NotificationPreferences" asp-action="SaveNotificationPreferences" asp-route-userType="@Model.UserType">
-  <div class="nhsuk-form-group">
-    <div class="nhsuk-fieldset" aria-describedby="notification-hint">
-      <legend class="nhsuk-fieldset__legend nhsuk-fieldset__legend--l">

```

Comment:
The legend doesn't do anything useful, no?

Reply:
It was invalid html because we weren't doing the fieldset properly (div is marked as nhsuk-fieldset). Maybe I should have instead changed the div to a fieldset? The page style is more consistent with the rest of the site now though

Reply:
Hmm. I guess the the `legend` came from the page originally being modeled on the nhs-checkboxes component. But I feel like we've overhauled the appearance of the component so much that it's not really an nhs-checkboxes element anymore (or a fieldset for that matter), so I agree with you, I don't think there's any point sticking with fieldset/legend.